modules/special-rules/pages/ork-initiative.adoc

```diff
@@ -0,0 +1,16 @@
+= Ork Initiative
+
+Always ready for a krumpin, Orks will often charge into melee before the enemy has even realised that the Orks are close enough to charge them!
+
+---
+
+Apply these rules to Ork armies:
```

Comment:
Flagging for future attention, this will need updating to whatever format is used for Tyranid Objectives.

Reply:
This appears to match the Tyranid Objectives format, marking as resolved.